nonos/logging.py

```diff
@@ -17,3 +19,18 @@ def print_err(message):
     https://github.com/neutrinoceros/idefix_cli
     """
     rprint(f":bone: [bold white on red]Error[/] {message}", file=sys.stderr)
+
+
+def logging_verbose(verbose):
```

Comment:
This took me much too long to decipher
```suggestion
def logging_verbose(verbose:int):
```